### PR TITLE
DataGrid: Fix column resizing when columnResizingMode is widget and rtlEnabled is true (T837344)

### PR DESCRIPTION
**Pull request by Alyar666**
See https://devexpress.com/issue=T837344

### CODE REVIEW

File: js/ui/grid_core/ui.grid_core.columns_resizing_reordering.js [hunk 1]
```diff
@@ -575,12 +575,17 @@ const ColumnsResizerViewController = modules.ViewController.inherit({
         return false;
     },
 
+    _isRtlMode: function() {
```
Comment (vconst) [inline js/ui/grid_core/ui.grid_core.columns_resizing_reordering.js]: _isRtlMode -> _isRtlParentStyle

Comment (vconst) [reply js/ui/grid_core/ui.grid_core.columns_resizing_reordering.js]: Also, I think that we should check this style only if rtlEnabled is true to improve performance without rtl

**Review by vconst — CHANGES_REQUESTED** [review]

File: js/ui/grid_core/ui.grid_core.columns_resizing_reordering.js [hunk 2]
```diff
@@ -733,7 +733,7 @@ const ColumnsResizerViewController = modules.ViewController.inherit({
             that._trackerView.show();
             const scrollable = that.component.getScrollable();
 
-            if(scrollable && that.option('rtlEnabled') && that._isRtlMode()) {
+            if(scrollable && that.option('rtlEnabled') && that._isRtlParentStyle()) {
```
Comment (jtoming830) [inline js/ui/grid_core/ui.grid_core.columns_resizing_reordering.js]: isRtlParentStyle will check for this.option('rtlEnabled') so check rtlEnabled here is unnecessary

File: js/ui/grid_core/ui.grid_core.columns_resizing_reordering.js [hunk 3]
```diff
@@ -575,17 +575,17 @@ const ColumnsResizerViewController = modules.ViewController.inherit({
         return false;
     },
 
-    _isRtlMode: function() {
-        return this._$parentContainer && this._$parentContainer.parent().css('direction') === 'rtl';
+    _isRtlParentStyle: function() {
+        return this._$parentContainer && this.option('rtlEnabled') && this._$parentContainer.parent().css('direction') === 'rtl';
```
Comment (jtoming830) [inline js/ui/grid_core/ui.grid_core.columns_resizing_reordering.js]: You can minify this condition using optional chaining. 
`this.option('rtlEnabled') && this._$parentContainer?.parent.().css.('direction') === 'rtl';`

**Review by vconst — APPROVED** [review]

**Pull request merged**